tests/unit/rql/query_builder/test_rql_dot_path.py

```
@@ -29,9 +29,9 @@ class Test:  # noqa: WPS431
 def test_dotted_path_comp_bool_and_str(op):
     result = getattr(RQLQuery().asset.id, op)
 
-    assert str(result("value")) == f"{op}(asset.id,value)"
-    assert str(result(True)) == f"{op}(asset.id,true)"  # noqa: FBT003
-    assert str(result(False)) == f"{op}(asset.id,false)"  # noqa: FBT003
+    assert str(result("value")) == f"{op}(asset.id,'value')"
+    assert str(result(True)) == f"{op}(asset.id,'true')"  # noqa: FBT003
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `FBT003`)

Remove unused `noqa` directive

(RUF100)